wellregistry/registry/admin.py

```diff
@@ -1,3 +1,78 @@
 from django.contrib import admin
+from django.utils.html import format_html
+from .models import Registry
 
-# Register your models here.
+# this is the Django property for the admin main page header
+admin.site.site_header = 'NGWMN Well Registry Administration'
+
+
+def check_mark(value):
+    """
+    This is a helper method to create an html formatted entry for the flags in tables.
+    """
+    return format_html('&check;') if value == 1 else ''
+
+
+class MultiDBModelAdmin(admin.ModelAdmin):
```

Comment:
Could you elaborate on what problem this class is supposed to solve?

Reply:
I suppose I could. I have detailed doc string describing it.

Reply:
Lets see.... When you have more than the 'default' database alias then it is required to set the 'using' value to the database alias to use in a db action. This ensures that the admin access to the registry table uses the connection alias with the appropriate granted access or db roles.

Reply:
I added the above comment to the class docstring

Reply:
The reason I asked didn't seem obvious to me from the definition of connection parameters that was a multi-database scenario as much as a multi-database user scenario.

Reply:
Sure, If I can make it clearer then by all means let me know. We could also perform an interactive pier review together. I would gladly update docstrings while we reviewed.